components/infra-proxy-service/server/nodes.go

```diff
@@ -115,9 +139,11 @@ func (s *Server) UpdateNode(ctx context.Context, req *request.UpdateNode) (*resp
 // roles: it would be 'roles:ROLE_NAME'
 // environments: it would be 'chef_environment:ENVIRONMENT_NAME'
 // policyfiles: it would be 'chef_environment:POLICY_FILES_GROUP_NAME'
-func (c *ChefClient) fetchAffectedNodes(ctx context.Context, chefType, name, version string) (*chef.SearchResult, error) {
+func (c *ChefClient) fetchAffectedNodes(ctx context.Context, url string) (*chef.SearchResult, error) {
```

Comment:
Small naming issue.  `url` here isn't really URL, it is the search query. And the variable we've called query is probably more accurately something like the "attrMap" or "attrFilter".

Reply:
Renamed `url` with `statement` and `query` with `params` seems matched with go-chef lib func signature https://github.com/go-chef/chef/blob/c278260aa167af1515bb3e2c282c2f02a2597543/search.go#L129-L138